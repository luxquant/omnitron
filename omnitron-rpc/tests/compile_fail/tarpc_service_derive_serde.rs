#[omnitron_rpc::service(derive_serde = loop {})]
trait World {
    async fn hello();
}

fn main() {}
