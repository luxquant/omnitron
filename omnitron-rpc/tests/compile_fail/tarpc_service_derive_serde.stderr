error: expected literal
 --> tests/compile_fail/omnitron_rpc_service_derive_serde.rs:1:33
  |
1 | #[omnitron_rpc::service(derive_serde = loop {})]
  |                                        ^^^^
